.husky/pre-commit

```diff
@@ -2,8 +2,8 @@
 . "$(dirname -- "$0")/_/husky.sh"
 echo "lint code"
 cd ./src/backend
-npm run precommit
+npm run lint && npx lint-staged
 
 cd ../frontend
 
-npm run precommit
+npm run lint && npx lint-staged && npm run lint:styles
```

Comment:
same as above. You should fix your local instead if it already works for all.

How do you commit btw? Have you added --no-verify?